nuxeo-core/nuxeo-core-event/src/main/java/org/nuxeo/ecm/core/work/api/WorkManager.java

```diff
@@ -267,7 +267,9 @@
      *            RUNNING}, or {@code null} for non-completed
      * @return the found work instance, or {@code null} if not found
      * @since 7.3
+     * @deprecated since 10.2 use {@link WorkManager#getWorkState(String)}
```

Comment:
but getWorkState is also deprecated, either we undeprecate the method either we explain that getWorkState is maintain for migration purpose only and its usage is discouraged.
We definitly need a NXDOC to explain the intend and help on code migration, this link could be added to the deprecated method.